.env

```diff
@@ -1 +1 @@
-REACT_APP_BACKEND_URL=http://localhost:3002
```

Comment:
.env files should not be pushed to github you must check you gitignore file

Reply:
This .env file is in all the other branches. And also the change is identical to the other branches so it's kinda useless that github desided to show this